code/modules/overmap/ships/computers/helm.dm

```diff
@@ -67,6 +67,8 @@
 	return 0
 
 /obj/machinery/computer/helm/attack_hand(var/mob/user as mob)
+	if (stat & NOPOWER)
+		return
```

Comment:
Same as above, there is already a check in the parent proc.

Reply:
can't find these check too in parent proc